src/main/java/com/google/jenkins/plugins/storage/DownloadStep.java

```diff
@@ -189,18 +189,20 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace,
           .replaceMacro(getLocalDirectory(), run, listener);
       FilePath dirPath = workspace.child(dirName);
 
-      List<StorageObject> objects = resolveBucketPath(bucketPath,
+      List<StorageObjectId> objects = resolveBucketPath(bucketPath,
           getCredentials(), version);
 
-      logger.info("Downloading " + objects.size() + " objects.");
+      listener.getLogger()
+          .println(module.prefix(
+              Messages.Download_FoundForPattern(objects.size(), path)));
 
-      DownloadSpec spec = new DownloadSpec(dirPath, objects);
+      DownloadSpec spec = new DownloadSpec(objects);
       // TODO(agoulti): add a download report.
 
       String resolvedPrefix = StorageUtil
           .replaceMacro(pathPrefix, run, listener);
 
-      initiateDownloadsAtWorkspace(getCredentials(), run, spec, dirPath,
+      initiateDownloadsAtWorkspace(getCredentials(), run, spec, dirPath, listener,
```

Comment:
This line (plus a few others in this file) generates a checkstyle error (longer than 80 chars).

Reply:
Yes, thanks. It seems Intellij has lost my settings over a restart :-(